### PR TITLE
Replace BitmapVisionView with VisionGLView

### PR DESCRIPTION
**Pull request by kiryldz**
Activity can not be run from UI, the only possibility to run it is:
adb shell am start -n com.mapbox.vision.teaser.debug/com.mapbox.vision.examples.activity.vision.VisionSampleActivity

* [ ] Rebase to `dev` branch
* [X] Assign reviewers

Linked issues:
[1579](https://app.zenhub.com/workspaces/vision-sdk-5ca52e6c027e3e7234ac7512/issues/mapbox/mapbox-vision/1579)

### CODE REVIEW

File: app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            VisionManager.create(customVideoSource)
+            VisionManager.start()
+            VisionManager.visionEventsListener = visionEventsListener
+            VisionManager.setVideoSourceListener(visionView)
+            visionManagerWasInit = true
+        }
+    }
+
+    private fun stopVisionManager() {
+        if (visionManagerWasInit) {
```
Comment (RingerJK) [inline app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt]: why do you need `visionManagerWasInit` flag?

File: app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+                videoSourceListener!!.onNewFrame(
+                    rgbaByteBuffer.array(),
+                    ImageFormat.RGBA,
+                    imageSize
+                )
+                rgbaByteBuffer.clear()
+            }
+        } catch (e: RuntimeException) {
+            e.printStackTrace()
```
Comment (RingerJK) [inline app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt]: `VisionLogger` here and in next catch

**Comment by yunikkk** [comment]
Seems it should be `DO NOT MERGE` or even draft PR, since it depends on GL view implemented in mapbox-vision

File: app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    // replace with your own path to video file
+    // private val PATH_TO_VIDEO_FILE = Environment.getExternalStorageDirectory().path + "/Telemetry/2019-08-27_09-48-52+0800/video0.mp4"
+    private val PATH_TO_VIDEO_FILE = "path_to_your_video_file"
+
+    private var videoSourceListener: VideoSourceListener? = null
+    private val handlerThread = HandlerThread("VideoDecode")
+    private lateinit var visionView: VisionView
+    private var visionManagerWasInit = false
+
+    private var mIsGlView = true
```
Comment (yunikkk) [inline app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt]: nit : remove `m` prefix

File: app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+        override fun detach() {
+            videoSourceListener = null
+            handlerThread.quitSafely()
+        }
+    }
+
+    // VisionEventsListener handles events from Vision SDK on background thread.
+    private val visionEventsListener = object : VisionEventsListener {
+
+        override fun onAuthorizationStatusUpdated(authorizationStatus: AuthorizationStatus) {}
```
Comment (yunikkk) [inline app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt]: No need to override empty funs - default implementation exists

File: app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    public override fun onCreate(savedInstanceState: Bundle?) {
+        if (savedInstanceState != null) {
+            mIsGlView = savedInstanceState.getBoolean("is_gl_view")
+        }
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_vision_sample)
+        if (mIsGlView) {
+            visionView = vision_gl_view
+            btn_switch_view.text = "Switch to Bitmap view"
+            vision_gl_view.visibility = VISIBLE
```
Comment (yunikkk) [inline app/src/main/java/com/mapbox/vision/examples/activity/vision/VisionSampleActivity.kt]: nit : use `show`/`hide`/`invisible` extensions

**Pull request merged**